src/components/AppHeader.tsx

```diff
@@ -34,13 +34,13 @@ const AppHeader: FC = ({ children }) => {
 
 export const HeaderDivider = styled.div`
   width: 1px;
-  height: 10px;
-  margin: 0 10px;
+  height: var(--spacing-10);
```

Comment:
**TLDR; The variable names are temporary and I think it's a good step towards a more consistent code-base.**

> adding a static value to the name ("10"), kinda cancel the abstraction... Updating the variable value would also require updating it's name.
At the same time, a more generic way to name variables (ie. spacing-small, big etc.) would reduce flexibility and granularity.

I agree with you 💯% @mvaivre. My reasoning is the following:
1. First, I had to become aware of what are the values that repeat in the code. To do that, I did global searches in the codebase and extracted common values that I found in CSS variables in a centralized location. The CSS variable names are just temporary.
2. The second step is to inspect these values and find patterns. Example patterns:
    - Are the values following a Fibonacci scale
    - Are the values following a linear 8px scale
    - etc
3. Third step would be (imo) to clean up these values and make sure they follow the scale we agree upon.
3. Then we can come up with proper naming, which I would suggest:
   - `spacing-1` for the smallest value of our scale
   - `spacing-2` for the immediately next one
   - `spacing-{X}` and so on

That way, when creating a new component and we want to give it some padding, for example, we can start with using a random value (say, `spacing-3`), see if it looks good, and if not, reduce/increase accordingly until it looks just right. No need to fiddle with px in that case anymore.

At the moment, when I want to create a new component, I need to look into other components to see what values are being used, so that I don't introduce yet another arbitrary value. It's a time-consuming process and does not endorse consistency.

Reply:
Yes, we had this discussion about the new website.

Honestly, I'm not 100% convinced just yet as I had a hard time tweaking the website.
Sometimes, some elements of the design need to be tweaked specifically, ie. you may want to use a certain spacing under a title (say, `spacing-1`) before realising you need to increase it by just a tiny bit (iterative design, which worked great for us so far!). In that case:

1) You can switch to `spacing-2`, but if your following a scale, it has to be granular enough. IMO, the linear scale used for the website was not, and that was kinda frustrating. So you may end up creating dozens of variables, which dramatically reduce the benefits of using variables...!

2) You can update the pixel value of `spacing-1` to increase it by a tiny bit. But in that case, all component using this variable will obviously see their spacing be increased by that amount, which may not be the expected behaviour. So you end up creating a new `spacing' variable, encountering the issue explained in point 1).

I feel like these styling variables makes a lot of sense for things like `border-radius`, `box-shadow`, or `font-weight` which are very transversal and where you want consistency. For paddings, margins etc., I would probably use a "component-based" approach : you define the styling precisely in the component (say, a generic container), then you reuse the component. 

But that's just my gut feeling based on a limited experience tweaking the website in a rush. 
All I want is to be wrong, so we can try this approach for a while, and see where it leaves us!

Reply:
I hear your points @mvaivre, thanks for sharing them!

How about we create only the following 5 variables, since they are the most frequent ones (see the frequency in the comments of following snippet) and name them based on a 5px spacial linear scale (where the step is 5px)?

```css
    --spacing-1: 5px;
    --spacing-2: 10px;
    --spacing-3: 15px;
    --spacing-4: 20px;
    --spacing-5: 25px;
```

```css
    --spacing-3: 3px;   // 3
    --spacing-5: 5px;   // 18
    --spacing-6: 6px;   // 3
    --spacing-7: 7px;   // 3
    --spacing-8: 8px;   // 2
    --spacing-10: 10px; // 24
    --spacing-12: 12px; // 5
    --spacing-15: 15px; // 26
    --spacing-16: 16px; // 2
    --spacing-20: 20px; // 22
    --spacing-25: 25px; // 16
    --spacing-30: 30px; // 2
    --spacing-40: 40px; // 5
    --spacing-46: 46px; // 3
    --spacing-50: 50px; // 6
    --spacing-60: 60px; // 9
    --spacing-80: 80px; // 1
```

Reply:
Thanks for hearing my points ;)

What would you do with the other spacings (those in between the 5px increments)?

Reply:
> What would you do with the other spacings (those in between the 5px increments)?

Delete the CSS variables and revert my changes.

Reply:
Keeping those 5 variables at least is an indication that when creating a new component we should first try to use those (since most of our components use them, hence promote a UI with similar spacings), and if not, then you're free to use whatever value you want in the component.

Reply:
Alright, works for me!